fedn/fedn/common/tracer/mongotracer.py

```diff
@@ -57,11 +57,9 @@ def new_session(self, id=None):
         data = {'session_id': str(id)}
         self.sessions.insert_one(data)
 
-    def new_round(self, id):
+    def new_round(self, round_data):
         """ Create a new session. """
-
-        data = {'round_id': str(id)}
-        self.rounds.insert_one(data)
+        self.rounds.insert_one(round_data)
```

Comment:
should there not be a check if round_id already exist?

Reply:
Yes, I added a TODO, plan to address it when refactoring mongotracer.